packages/web/src/components/gcds-fieldset/gcds-fieldset.tsx

```diff
@@ -49,259 +20,53 @@ export class GcdsFieldset {
    */
   @Prop({ reflect: true, mutable: false }) fieldsetId!: string;
 
-  /**
-   * The title for the contents of the fieldset
-   */
-  @Prop({ reflect: true, mutable: false }) legend!: string;
-
-  /**
-   * Flag the contents are required
-   */
-  @Prop({ reflect: true, mutable: false }) required: boolean;
-
-  /**
-   * Error message for invalid form elements in group.
-   */
-  @Prop({ reflect: true, mutable: true }) errorMessage: string;
-  @Watch('errorMessage')
-  validateErrorMessage() {
-    if (this.disabled) {
-      this.errorMessage = '';
-    } else if (!this.hasError && this.errorMessage) {
-      this.hasError = true;
-    } else if (this.errorMessage == '') {
-      this.hasError = false;
-    }
-  }
-
   /**
    * Hint displayed below the legend.
    */
   @Prop({ reflect: true, mutable: false }) hint: string;
 
   /**
-   * Flag to disable fieldset and its contents
+   * The title for the contents of the fieldset
    */
-  @Prop({ reflect: true, mutable: true }) disabled: boolean;
-
-  @Watch('disabled')
-  validateDisabledFieldset() {
-    if (this.required) {
-      this.disabled = false;
-    }
-
-    if (this.disabled == true) {
-      for (let i = 0; i < this.el.children.length; i++) {
-        this.el.children[i].setAttribute('disabled', '');
-      }
-    }
-  }
-
-  @Watch('disabled')
-  handleDisabledChange(newValue: boolean, _oldValue: boolean) {
-    if (_oldValue && newValue != _oldValue) {
-      for (let i = 0; i < this.el.children.length; i++) {
-        this.el.children[i].removeAttribute('disabled');
-      }
-    }
-  }
+  @Prop({ reflect: true, mutable: false }) legend!: string;
 
   /**
-   * Array of validators
+   * Sets the appropriate font size for the fieldset legend.
    */
-  @Prop({ mutable: true }) validator: Array<
-    string | ValidatorEntry | Validator<string>
-  >;
+  @Prop({ mutable: true }) size!: 'h2' | 'h3' | 'h4' | 'h5' | 'h6';
```

Comment:
@daine @ethanWallace do we want to keep this prop name as `size` or should we rename it to `legend-size` to make it very clear that it only changes the size of the legend?

Reply:
`legend-size` would be a more descriptive attribute name.

Reply:
@daine what do you think?